test/main.cpp

```diff
@@ -7,109 +7,109 @@
 #include <iostream>
 
 TEST_CASE("Operator==") {
-    r2d2::robot_arm::coordinate_3D_c c(1, 5, 6);
-    r2d2::robot_arm::coordinate_3D_c c2(1, 5, 6);
+    r2d2::robot_arm::vector3_c<float> c(1.f, 5.f, 6.f);
+    r2d2::robot_arm::vector3_c<float> c2(1.f, 5.f, 6.f);
     REQUIRE(c == c2);
 }
 
 TEST_CASE("Default constructor") {
-    r2d2::robot_arm::coordinate_3D_c c;
-    REQUIRE(c == r2d2::robot_arm::coordinate_3D_c(0, 0, 0));
+    r2d2::robot_arm::vector3_c<int> c;
```

Comment:
maybe make a typedef for vector3_c type int as its frequently used?

Reply:
Added typedef